content/documentation/pages/2-concepts/6-monitoring.md

```diff
@@ -0,0 +1,29 @@
+---
+path: 'concepts/monitoring/'
+title: 'Monitoring'
```

Comment:
I'd suggest removing this section entirely. I ported the entire content to feature guide yday. This duplicates that content.